src/System.Data.Common/tests/System.Data.Common.Tests.csproj

```diff
@@ -104,6 +104,7 @@
     <Compile Include="System\Data\VersionNotFoundException.cs" />
     <Compile Include="System\Data\XmlDataLoaderTest.cs" />
     <Compile Include="System\Data\XmlDataReaderTest.cs" />
+    <Compile Include="System\Xml\XmlDataDocumentTests.cs" />
```

Comment:
The build will fail for uap since it will build against netstandard and this type is not exposed there. You should conditionally include this test to when TargetGroup != netstandard